### DISCUSSION TITLE
Can we delete Shell Command search history?

### DISCUSSION THREAD

**Discussion by yfzm** [Flow-Launcher/Flow.Launcher#932] [Ideas]
When using **Shell Command** plugin, I had executed a wrong command, and it seems the record will always show up and there is no simple way to delete it (maybe a little OCD :joy:). Adding a delete option when right-clicking the item will be great.

**Comment by jjw24** [comment]
U can manually delete it for now, the records are kept in user data -> settings -> shell plugin.

Let me know if you can't find it

**Reply by yfzm** [reply]
Yes, I found it and deleted the record and it works. However, it is not easy because it didn't work when I deleted the historical record while Flow Launcher is running. In that case, the record would be written back when the Launcher is exiting.

**Reply by jjw24** [reply]
Yes forgot to mention you need to quit flow, because this records r loaded in memory, so they would be written back to the file.
